### PR TITLE
Hide binary file contents

### LINKED ISSUE #1397: Server error when focus on PDF bundle
```
2015-11-12 11:41:25,721 ERROR Internal Server Error: /api/bundles/0xe2fa97ebd721418db5e589e0a32ff86f/
Traceback (most recent call last):
  File "/Users/sckoo/Developer/codalab/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 138, in get_response
    response = response.render()
  File "/Users/sckoo/Developer/codalab/venv/lib/python2.7/site-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/Users/sckoo/Developer/codalab/venv/lib/python2.7/site-packages/rest_framework/response.py", line 59, in rendered_content
    ret = renderer.render(self.data, media_type, context)
  File "/Users/sckoo/Developer/codalab/venv/lib/python2.7/site-packages/rest_framework/renderers.py", line 81, in render
    indent=indent, ensure_ascii=self.ensure_ascii)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 250, in dumps
    sort_keys=sort_keys, **kw).encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xe4 in position 10: invalid continuation byte
[12/Nov/2015 11:41:25] "GET /api/bundles/0xe2fa97ebd721418db5e589e0a32ff86f/?_=1447328479938 HTTP/1.1" 500 18966
```

Causes the bundle permissions to fail to load:
screen shot 2015-11-12 at 3 42 41 am

### PR DESCRIPTION
**Pull request by kashizui**
Prevents crashing on binary text

Fixes #1397 

@percyliang @pbhatnagar3 please review

looks like this on the side panel:
screen shot 2015-11-16 at 5 45 19 pm

### CODE REVIEW

File: codalab/apps/web/bundles.py [hunk 1]
```diff
@@ -202,32 +203,35 @@ def upload_bundle(self, source_file, bundle_type, worksheet_uuid):
...
             For each file, only return the first few lines.
-            '''
+            """
             def get_lines(name):
                 lines = self.head_target((uuid, name), self.HEAD_MAX_LINES)
                 if lines is not None:
                     import base64
                     lines = ''.join(map(base64.b64decode, lines))
-                return lines
+                try:
```
Comment (percyliang) [inline codalab/apps/web/bundles.py]: Maybe we should move this to the codalab-cli repo and have it be a utility function that both the BundleCLI and this file can call.

Comment (kashizui) [inline codalab/apps/web/bundles.py]: Added this functionality in `formatting.verbose_contents_str`

**Comment by percyliang** [comment]
Nice, thanks!  LGTM

File: codalab/apps/web/bundles.py [hunk 2]
```diff
@@ -202,32 +203,33 @@ def upload_bundle(self, source_file, bundle_type, worksheet_uuid):
...
-            '''
+            """
             def get_lines(name):
                 lines = self.head_target((uuid, name), self.HEAD_MAX_LINES)
                 if lines is not None:
                     import base64
                     lines = ''.join(map(base64.b64decode, lines))
-                return lines
+
+                return formatting.verbose_contents_str(lines)
```
Comment (kashizui) [inline codalab/apps/web/bundles.py]: Using formatting helper now.

@percyliang please review

**Pull request merged**